docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
 # Computational Infrastructure
 
-## Introduction
-One way of understanding the Inductiva API is to see it as an abstraction layer that
-allows you to use computational resources made available by different providers in a
-seamless and uniform fashion. 
+One way to understand the Inductiva API is to see it as an intermediary between 
+users and the complex landscape of computational resources. In this reference, you 
+will learn how the Inductiva API manages your computational workload, and then you'll
+explore the various computational options currently available to you, and additional resources set to be introduced in future updates.
 
-On the server side, Inductiva takes a computational load issued by the user, i.e.,
-one or several simulations, and routes it to a computational resource that the users
-has access to, and takes care of all the orchestration required for running the load
-and returning the results back to the user. 
+## Computational Resource Management
 
-The computational resources effectively used to process that load can potentially be
-provided by a number of different players, and it is Inductiva’s role to facilitate
-the access to those resources, and even help you find the best one for your particular
-case (e.g. by price or by performance). 
+The Inductiva API acts as an abstraction layer that enables you to access a wide 
+array of computational resources across different providers through a unified 
+interface called the [Command Line Interface (CLI)](). The API's key role is to 
```

Comment:
I really like this description here,  but the interface is not the CLI, it is the Python Code that provides the abstraction layer.